packages/app/src/files-manager/components/files-manager/files-manager.interface.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { Type } from "Renderer/components/core/icon/icon.config"
+import { FilesType } from "App/files-manager/constants/files-manager.enum"
+
+export interface DiskSpaceCategory {
```

Comment:
```suggestion
export interface DiskSpaceCategory extends ChartItem {
```

It's too big an effort to adapt ChartItem (`megabyteSize` / `value` conflict) but in my opinion, this should be a direction to refactor (to avoids mapping)

Reply:
`StackedBarChart` is a reusable component so it should be ready for any value, not only for size 😉